crates/bevy_transform/src/components/transform.rs

```diff
@@ -120,6 +120,11 @@ impl Transform {
 
     /// Returns this [`Transform`] with a new rotation so that [`Transform::forward`]
     /// points towards the `target` position and [`Transform::up`] points towards `up`.
+    ///
+    /// In some cases it's not possible to construct a rotation. Another axis will be picked in those cases:
```

Comment:
I wonder if this means we should be returning `Result<Self, SomeError>` instead, seems odd to silently fail here.

Reply:
IMO we're not silently failing: we're performing a "best-attempt". I'd be open to `try` variants too.

Reply:
I would prefer to work by default, instead of having to add unwrap / error management everywhere this pops up